api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSourceInterceptorContext.java

```diff
@@ -0,0 +1,9 @@
+package org.eclipse.microprofile.config.spi;
+
+public interface ConfigSourceInterceptorContext {
+    String getKey();
```

Comment:
We don't want to lock the key in here, otherwise we can't use an interceptor for property evaluation (which has to query potentially several property names).